testing-mysql-server-5/repack-mysql-5.sh

```diff
@@ -62,6 +67,21 @@ $TAR -czf $OLDPWD/$RESOURCES/mysql-Linux-amd64.tar.gz \
 popd
 rm -rf $PACKDIR
 
+PACKDIR=$(mktemp -d "${TMPDIR:-/tmp}/mysql.XXXXXXXXXX")
+cp $LINUX_PPC64LE_DIST $PACKDIR/
+pushd $PACKDIR
+rpm2cpio $LINUX_PPC64LE_RPM | cpio -idm
+mkdir -p mysql-Linux-ppc64le/bin mysql-Linux-ppc64le/lib64 mysql-Linux-ppc64le/share mysql-Linux-ppc64le/data
+cp usr/bin/mysql_install_db mysql-Linux-ppc64le/bin/
```

Comment:
can you explain what these cps are for?

Reply:
The directory organization inside the binary dist tarball used for intel matches the one expected by this repo. e.g. [this](https://github.com/amitsadaphule/testing-mysql-server/blob/ppc64le_support/testing-mysql-server-base/src/main/java/com/facebook/presto/testing/mysql/AbstractEmbeddedMySql.java#L128) line expects:
rootdir
  |- bin
     | - mysqld
     |- ...
So, the intel repack directly provides the paths of the necessary files to tar command [here](https://github.com/amitsadaphule/testing-mysql-server/blob/ppc64le_support/testing-mysql-server-5/repack-mysql-5.sh#L58).

The rpm used for ppc64le however has a different directory structure. So, I had to manually create the expected directory structure and copy the necessary files to it from the extracted rpm.